Bump coverlet version

MarcoRossignoli opened the pull request:
closes https://github.com/dotnet/test-templates/issues/77

Upgrade to last version released a week ago https://www.nuget.org/packages/coverlet.collector/ 10K download and no issue on repo for now

If it's too brave we can update with `1.1.0` 433K download, however we can always tell to user to downgrade in case.

Fixed searching for `findstr /n /s /c:"coverlet.collector"  *proj`

/cc: @singhsarab  @nohwnd @shyamnamboodiripad @AbhitejJohn

singhsarab commented:
@nohwnd Once this goes in, we can probably go ahead and make insertion for the templates as well

AbhitejJohn commented:
@MarcoRossignoli : That's good indeed. Noticed though that we use the latest version of the Test SDK. We'd probably want to have a test that validates with the version that ships in this template. We could definitely keep you in loop when we ship a new version (@nohwnd , it would be great if you could add this to the release steps.) or update the template here. It would be nice to have this automated for sure. I haven't really used dependabot - how does it integrate with code/build pipeline updates.

MarcoRossignoli commented:
> We'd probably want to have a test that validates with the version that ships in this template

Ok if we want to test that it works with template sdk I can update test to test for every sdk shipped in template, I'll work on it. In this case cannot use dependabot but it's not a problem.

MarcoRossignoli commented:
Ok we've multi sdk tests https://github.com/tonerdo/coverlet/pull/687

MarcoRossignoli commented:
Can we merge this guy @nohwnd?

MarcoRossignoli commented:
any news?

nohwnd commented:
@MarcoRossignoli I actually did not have permissions to merge anything here. Is the 1.1.1 still good to go? No influx of bugs for that version?

MarcoRossignoli commented:
We fixed a lot of edge case and bugs in last two release https://github.com/tonerdo/coverlet/blob/master/Documentation/Changelog.md from 1.1.0 onwards, I mean users can use this version and upgrade lib if needed(if they notice new one). It's not mandatory, but  from time to time we should bump a bit, or we should understand how to handle this workflow.
Coverage needs sometimes to run after new core(roslyn) release to catch new il changes cases.

nohwnd commented:
> Upgrade to last version released a week ago https://www.nuget.org/packages/coverlet.collector/ 10K download and no issue on repo for now 

Sorry, I was just reacting to this comment (and got the version wrong). Let me re-iterate.  :)

Do you still feel that 1.2.0 is good stable release as you did when creating this PR? Say yes and I will merge 🙂

MarcoRossignoli commented:
BTW more importand is test sdk because when inproc collectors fails to load no warning/error show to user and they could hit "100ms kill" I've just updated guide https://github.com/tonerdo/coverlet/blob/master/Documentation/VSTestIntegration.md

> Do you still feel that 1.2.0 is good stable release as you did when creating this PR? Say yes and I will merge 🙂

Hehe 😃 you know how our work is...I would say yes 1 month out and no issue on it, I fixed some CI on users repo with this new version and no complain so I feel ok, btw in case of issue we'll ask to user to downgrade ver, do you have some internal "rule" to decide when release stable version?I know that depends on project but I'd like to know.

![image](https://user-images.githubusercontent.com/7894084/73366852-e4e24c00-42ae-11ea-91f8-69fa158f3749.png)

MarcoRossignoli commented:
So yes!

MarcoRossignoli commented:
Thanks @nohwnd!
